components/script/dom/xmlhttprequest.rs

```diff
@@ -1038,6 +1052,25 @@ impl XMLHttpRequest {
         encoding.decode(&self.response.borrow(), DecoderTrap::Replace).unwrap().to_owned()
     }
 
+    fn blob_response(&self) -> Root<Blob> {
```

Comment:
Spec link above this function, please. (https://xhr.spec.whatwg.org/#blob-response)